scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -313,6 +313,13 @@ object Decl {
       tparams: List[scala.meta.Type.Param],
       bounds: scala.meta.Type.Bounds
   ) extends Decl with Member.Type
+  @ast class Given(
+      mods: List[Mod],
+      name: scala.meta.Name,
+      tparams: List[scala.meta.Type.Param],
+      sparams: List[List[Term.Param]],
+      decltpe: scala.meta.Type
+  ) extends Defn
```

Comment:
extends Decl

Reply:
Fixed! That was some bad copy paste :sweat: